calcom/cal.com
# fix: Able to toggle off timeFormat on new booker UI

## What does this PR do?

The new booking UI had two bugs that I spotted whilst debugging another piece of work:

- Clicking the already-selected layout caused the page to break (already fixed with 8755ed0c5e7fb61303a1d1fd7d54649c0e28cd0b)
- Clicking the already-selected time format removed time formatting

This PR fixes that second issue by, as per Radix Toggle Group docs, [ensuring there is always a value](https://www.radix-ui.com/docs/primitives/components/toggle-group#ensuring-there-is-always-a-value).



Fixes (no issue exists for this)

https://www.loom.com/share/911627ee74b846849e8014ece062ed22

## Type of change



- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Enable the new booking UI
- Visit a users booking page
- Select the monthly view
- Choose a day
- Test bug fix: On the "Available Times" section, click the already-selected time format
  - Before
    - [x] See all time formats unselected
    - [x] See all rendered times looking very ugly: 
  - After
    - [x] No change
- Regression test: Swap between 12h and 24h a few times
  - [x] See the time format change as expected

## Mandatory Tasks

## Checklist

(all completed)
